src/clients/c++/perf_client/perf_client.cc

```diff
@@ -259,142 +275,293 @@ Usage(char** argv, const std::string& msg = std::string())
+                   "abort the execution once the measured latency exceeds this "
+                   "threshold. By default, latency threshold is set 0 and "
+                   "the perf_client will run for entire --search-range.",
+                   18)
+            << std::endl;
   std::cerr
-      << "For -s, it indicates the deviation threshold for the measurements. "
-         "The measurement is considered as stable if the recent 3 measurements "
-         "are within +/- (deviation threshold)% of their average in terms of "
-         "both infer per second and latency. Default is 10(%)"
+      << FormatMessage(
+             " --max-threads: Sets the maximum number of threads that will be "
+             "created for providing desired concurrency. Default is 16.",
+             18)
       << std::endl;
   std::cerr
-      << "For -c, it indicates the maximum number of concurrent requests "
-         "allowed if -d flag is set. Once the number of concurrent requests "
-         "exceeds the maximum, the perf client will stop and exit regardless "
-         "of the latency threshold. Default is 0 to indicate that no limit is "
-         "set on the number of concurrent requests."
+      << FormatMessage(
+             " --stability-offset (-s): It indicates the deviation threshold "
+             "for the measurements. The measurement is considered as stable if "
+             "the recent 3 measurements are within +/- (deviation threshold)% "
+             "of their average in terms of both infer per second and latency. "
+             "Default is 10(%).",
+             18)
       << std::endl;
+  std::cerr << FormatMessage(
+                   " --max-trials (-r): It indicates the maximum number of "
+                   "measurements for each profiling setting. The perf client "
+                   "will take multiple measurements and report the measurement "
+                   "until it is stable. The perf client will abort if the "
+                   "measurement is still unstable after the maximum number of "
+                   "measuremnts. The default value is 10.",
+                   18)
+            << std::endl;
   std::cerr
-      << "For -p, it indicates the time interval used for each measurement."
-      << " The perf client will sample a time interval specified by -p and"
-      << " take measurement over the requests completed"
-      << " within that time interval." << std::endl;
-  std::cerr << "For -r, it indicates the maximum number of measurements for "
-               "each profiling setting. The perf client will take multiple "
-               "measurements and report the measurement until it is stable. "
-               "The perf client will abort if the measurement is still "
-               "unstable after the maximum number of measuremnts."
+      << FormatMessage(
+             " --percentile: It indicates that the specified percentile in "
```

Comment:
"Indicates the confidence value as a percentile that will be used to determine if a measurement is stable. For example, a value of 85 indicates that the 85th percentile latency will be used to determine stability. The percentile will also be reported in the results. The default is -1 indicating that the average latency is used to determine stability"